.github/workflows/ci.yml

```diff
@@ -15,9 +15,9 @@ jobs:
           os: ubuntu-latest
         - build: x86_64-macos
           os: macos-latest
+          target: x86_64-apple-darwin
         - build: aarch64-macos
           os: macos-latest
-          target: aarch64-apple-darwin
```

Comment:
Should we leave this in for good measure?

Reply:
Sure, can't hurt!